pipelines/response-operations-social-ui.yml

```diff
@@ -0,0 +1,206 @@
+---
+resource_types:
+- name: slack-notification
+  type: docker-image
+  source:
+    repository: cfcommunity/slack-notification-resource
+
+- name: cf-cli-resource
+  type: docker-image
+  source:
+    repository: nulldriver/cf-cli-resource
+    tag: latest
+
+resources:
+- name: response-operations-social-ui-source
+  type: git
+  source:
+    uri: https://github.com/ONSdigital/response-operations-social-ui.git
+    branch: master
+
+- name: ras-deploy
+  type: git
+  source:
+    uri: https://github.com/ONSdigital/ras-deploy.git
+    branch: master
+
+- name: cf-resource-preprod
+  type: cf
+  source:
+    api: ((preprod_cloudfoundry_api))
+    username: ((preprod_cloudfoundry_email))
+    password: ((preprod_cloudfoundry_password))
+    organization: rmras
+    space: preprod
+    skip_cert_check: true
+
+- name: cf-cli-resource-preprod
+  type: cf-cli-resource
+  source:
+    api: ((preprod_cloudfoundry_api))
+    username: ((preprod_cloudfoundry_email))
+    password: ((preprod_cloudfoundry_password))
+    org: rmras
+    space: preprod
+    skip_cert_check: true
+
+- name: cf-cli-resource-prod
+  type: cf-cli-resource
+  source:
+    api: ((prod_cloudfoundry_api))
+    username: ((prod_cloudfoundry_email))
+    password: ((prod_cloudfoundry_password))
+    org: rmras
+    space: prod
+    skip_cert_check: true
+
+- name: cf-resource-prod
+  type: cf
+  source:
+    api: ((prod_cloudfoundry_api))
+    username: ((prod_cloudfoundry_email))
+    password: ((prod_cloudfoundry_password))
+    organization: rmras
+    space: prod
+    skip_cert_check: true
+
+- name: notify
+  type: slack-notification
+  source:
+    url: ((slack_webhook))
+
+jobs:
+- name: response-operations-social-ui-preprod-deploy
+  serial: true
+  plan:
+  - get: response-operations-social-ui-source
+    trigger: true
+  - get: ras-deploy
+  - task: run-unit-tests
+    file: ras-deploy/tasks/python-unit-tests.yml
```

Comment:
Not sure what the agreed approach is for the team, but we could maybe put this in the r-ops-social repo instead? That way, ras-deploy as a resource would no longer be needed in here. This has been done in RH https://github.com/ONSdigital/respondent-home-ui/tree/master/ci. Not sure what's for the best though...